notes/lecture25.tex

```diff
@@ -0,0 +1,180 @@
+\filldraw[red] (0.5,-2) circle (2pt) node[anchor=north] {$x^*$};
+\draw[blue, thin] (0,0) .. controls (-0.5,-1) .. (0.5,-2);
+\draw[red, thin] (0,0) -- (-0.5,-0.5) -- (-0.1,-0.8) -- (-0.4,-1.1) -- (0.05,-1.4) -- (-0.05,-1.6) -- (0.5,-2);
+\end{tikzpicture}
+\caption{The central path\label{fig:central_path}}
+\end{figure}
+
+Our goal is to design an algorithm that will approximately follow the central path. Assume that we already get a "good" enough initial point, then at every step, we apply one step of Newton's method. To guarantee that the algorithm converges, we need to answer the following two questions:
+\begin{itemize}
+\item Under what conditions that the single-step Newton method works?
+\item How to carefully update $\epsilon$?
+\end{itemize}
+
+\subsubsection{Newton Decrement}
+To apply the Newton's method, first we need to find out the first-order and second-order derivatives of $f_\epsilon$. Note that
+\begin{eqnarray}
+\triangledown f_\epsilon(x)&=&c-\epsilon \sum_{j=1}^m \dfrac{A_j}{A^T_j x-b}\triangleq c-\epsilon A^T S^{-1}\mathbb{1}\label{eq:grad_f_eps}\\
+\triangledown^2f_\epsilon(x)&=&\epsilon A^TS^{-2}A=\epsilon \sum_{j=1}^m\dfrac{A_j A_j^T}{S_j^2}\label{eq:hessian_f_eps}
+\end{eqnarray}
+where $\mathbb{1}=[1,1,\cdots,1]^T\in\mathbb{R}^{m\times1}$, $S=diag\{A^T_1 x-b, A^T_2 x-b, \cdots, A^T_m x-b\}$.
+
+Then the Newton's update can be applied:
+\begin{equation}
+\bar{x}=x-[\triangledown^2f_\epsilon(x)]^{-1}\triangledown f_\epsilon(x)=x-[\epsilon A^TS^{-2}A]^{-1}\left(c-\epsilon A^T S^{-1}\mathbb{1}\right)\label{eq:Newton_update}
+\end{equation}
+
+Recall that the Newton's method finds the solution by making the first-order condition zero. To measure how much the Newton update will decrease the first-order approximation, we introduce the concept of Newton decrement.
+
+Define the Newton decrement as
+\begin{equation}
+q^2(x, \epsilon) = \triangledown f^T_\epsilon(x)[\triangledown^2f_\epsilon(x)]^{-1}\triangledown f_\epsilon(x)\label{eq:newton_decrement}
+\end{equation}
+or $q(x, \epsilon) = [\triangledown^2f_\epsilon(x)]^{-1/2}\triangledown f_\epsilon(x)$. 
+
+Note that the Newton decrement also relates to the difference between $f_\epsilon(x)$ and the minimum of its second-order approximation:
+\begin{eqnarray}
+&&f_\epsilon(x)-\min_{\bar{x}}\left(f_\epsilon(x)+\triangledown f_\epsilon(x)^T(\bar{x}-x)+(\bar{x}-x)^T\triangledown^2 f_\epsilon(x)(\bar{x}-x)\right)\nonumber\\
+&=&f_\epsilon(x)-\left(f_\epsilon(x)-\dfrac{1}{2}\triangledown f^T_\epsilon(x)[\triangledown^2f_\epsilon(x)]^{-1}\triangledown f_\epsilon(x)\right)\nonumber\\
+&=&\dfrac{1}{2}\triangledown f^T_\epsilon(x)[\triangledown^2f_\epsilon(x)]^{-1}\triangledown f_\epsilon(x)\triangleq \dfrac{1}{2}q^2(x, \epsilon).
+\end{eqnarray}
+
+We now will use the Newton decrement to answer the above two questions: conditions for single-step Newton to work and rules for updating $\epsilon$.
+
+\subsubsection{Propositions, Algorithms and Convergence Theorem}
```

Comment:
Start this subsection with a sentence about what's going to happen. Perhaps also change the section tilte to something more like "A convergence guarantee"